components/module-components/blog/PublishArticleForm.tsx

```diff
@@ -10,7 +10,11 @@ import TiptapEditor from '../../editor/TiptapEditor';
 import TagManager from './TagManager';
 import Button from '../../reusable-components/button/Button';
 
-const PublishArticleForm = () => {
+interface PublishArticleFormProps {
+  userRole: 'content_manager' | 'admin';
```

Comment:
we should already have some enum containing the roles for the user, please reuse that one. In case we don't have it, please create it.